### PR TITLE
version increase to 0.1.1

### PR DESCRIPTION
**Pull request by psychocoderHPC**
- version increase
- update changelog

- [x] wait for #90

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+0.1.1 "Colonel Worf"
+--------------------
+**Date:** 2019-01-23
+
+- update Alpaka submodule to 0.3.5 #81
+- add `cuplaGetErrorString` # 84
```
Comment (ax3l) [inline CHANGELOG.md]: little space too much

Comment (psychocoderHPC) [reply CHANGELOG.md]: :space_invader:

**Comment by psychocoderHPC** [comment]
please wait with the merge, I need to add a fix and append it in the changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+0.1.1 "Colonel Worf"
```
Comment (ax3l) [inline CHANGELOG.md]: Great that you credit Worf's grand dad

**Comment by psychocoderHPC** [comment]
good catch thx!

**Pull request merged**